documentation/antora.yml

```diff
@@ -1,6 +1,8 @@
 name: reference
 title: Debezium Documentation
-version: '1.7'
+version: 'dev'
```

Comment:
With this change, will we then publish _only_ to the .../dev/ URL, but to .../dev/ _and_ 1.7?

Reply:
We will publish only .../dev/ in this case.